packages/adapter-node/src/handler.js

```diff
@@ -45,7 +45,7 @@ const ssr = async (req, res) => {
 		return res.end(err.reason || 'Invalid request body');
 	}
 
-	setResponse(res, await app.render(request));
+	setResponse(res, await app.render({ request, platform: { nodeRequest: req } }));
```

Comment:
personally, I would just call this `request` as I think `platform.request` should hopefully be clear enough
```suggestion
	setResponse(res, await app.render({ request, platform: { request: req } }));
```

we should also update the README for each adapter to document what they populate into `platform` (i.e. `request` here and `env` in `adapter-cloudflare`)

Reply:
That makes sense, I may call it req instead though as that is what most node devs are used to

Reply:
I still like `request` more than `req`

Reply:
Done :)